src/Metadata/class-post-builder.php

```diff
@@ -475,7 +484,7 @@ private function get_author_name( ?WP_User $author ): string {
 			return '';
 		}
 
-		if ( ! empty( $author->display_name ) ) {
+		if ( '' !== $author->display_name ) {
```

Comment:
If there is no value then by default we have `''` so therefore just checking this.